Makefile

```diff
@@ -44,7 +44,13 @@ help:
 
 .PHONY: install-requirements
 install-requirements:
-	pipx install black flake8 argcomplete wheel huggingface_hub codespell
+	pipx install \
+			black~=25.0 \
```

Comment:
Is Renovate going to update these?

Reply:
I don't think so. 
I've searched for an option to use a `requirements.txt` or similar, which could be updated by renovate with pipx. I couldn't find anything, unfortunately. And it seems there will not be any support for this in pipx: https://github.com/pypa/pipx/issues/359